extra/contrib/fusion_rgb_cycle.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+import liquidctl.cli as lc
+import liquidctl
+import coloraide as c
+import argparse
+
+import time
+import sys
+
+teal = c.Color("#2de544")
+bluer = c.Color("#2d8a8d")
+deep_blue = c.Color("#001030")
+purple = c.Color("#c000c0")
+red = c.Color("#d00000")
+
+# set up argparse
+parser = argparse.ArgumentParser(
+    description="Cycle through colors on an Gigabyte RGB Fusion 2.0 device."
+)
+parser.add_argument(
+    "--space",
+    type=str,
+    default="srgb",
+    help="Color space to use; see https://facelessuser.github.io/coloraide/colors/",
+)
+# you can provide 0 or more colors to cycle through
+parser.add_argument("colors", nargs="*", help="Colors to cycle through")
+parser.add_argument("--debug", action="store_true", help="Print debug messages")
+parser.add_argument(
+    "--channel",
+    type=str,
+    default="led6",
+    help="Channel to use; see https://github.com/liquidctl/liquidctl/blob/main/docs/gigabyte-rgb-fusion2-guide.md",
+)
+parser.add_argument("--speed", type=float, default=1.0, help="Speed (seconds per color transition)")
+
+args = parser.parse_args()
+
+colors = [c.Color("#" + x) for x in args.colors]
+if len(colors) < 2:
+    colors.append(teal)
+if len(colors) < 2:
+    colors.append(bluer)
+
+devs = liquidctl.find_liquidctl_devices()
+for dev in devs:
+    if "Gigabyte RGB Fusion 2.0" in dev.description:
+        break
```

Comment:
If you're looking for a specific device class, you can use `RgbFusion2.find_supported_devices()`, which will only enumerate the required bus and only return devices of the specified class.

If on top of that you still need to do the equivalent of `--match` on the command line, you can pass `match="<prefix>"` to that method (`find_liquidctl_devices()` also supports it).

Reply:
Okay, for now I'm just going to use the first of these RGBFusion devices but if you have a recommended way to select one if the system has several, we could use that.